libpod/networking_linux.go

```diff
@@ -225,21 +226,55 @@ func (r *Runtime) setupRootlessNetNS(ctr *Container) error {
 
 	isSlirpHostForward := false
 	disableHostLoopback := true
+	var outboundAddr string
+	outboundAddr = nil
+
+	var outboundAddr6 string
+	outboundAddr6 = nil
+
 	if ctr.config.NetworkOptions != nil {
 		slirpOptions := ctr.config.NetworkOptions["slirp4netns"]
 		for _, o := range slirpOptions {
 			switch o {
 			case "port_handler=slirp4netns":
 				isSlirpHostForward = true
+				continue
 			case "port_handler=rootlesskit":
 				isSlirpHostForward = false
+				continue
 			case "allow_host_loopback=true":
 				disableHostLoopback = false
+				continue
 			case "allow_host_loopback=false":
 				disableHostLoopback = true
+				continue
+			}
+			// Parametrized options
+			switch {
+			case slirpFeatures.HasOutboundAddr && strings.Contains(o, "outbound_addr="):
```

Comment:
We should merge these two switch statements - there's no real reason to have two. Probably want to split O on `=` and do a comparison on the first element.

Reply:
merged